src/braket_container.py

```diff
@@ -91,6 +99,12 @@ def create_symlink():
             log_failure_and_exit(f"Symlink failure.\n Exception: {e}")
 
 
+def get_s3_client():
+    if not hasattr(_local, 's3_client'):
+        _local.s3_client = boto3.client("s3")
```

Comment:
The low-level client is already thread safe but this checks locally we have the client ready.